app/page.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { getSpotlights } from "@/data/drupal/spotlight";
+import { twJoin } from "tailwind-merge";
+import { Layout, LayoutContent } from "@uoguelph/react-components/layout";
+import { Header } from "@uoguelph/react-components/header";
+import { Footer } from "@uoguelph/react-components/footer";
+import { Container } from "@uoguelph/react-components/container";
+import { Typography } from "@uoguelph/react-components/typography";
+import { TagLine } from "@/components/client/home/tag-line";
+import { SpotlightHero } from "@/components/client/home/spotlight-hero";
+import { SpotlightCards } from "@/components/client/home/spotlight-cards";
+import { StudyHere } from "@/components/client/home/study-here";
+import { Rankings } from "@/components/client/home/rankings";
+import { ThreeCampuses } from "@/components/client/home/three-campuses";
+import { HomeStory } from "@/components/client/home/story";
+
+export default async function Page() {
+  const { hero, cards } = await getSpotlights();
+  const containerClasses = twJoin("pt-6");
+
+  return (
+    <Layout>
+      <Header></Header>
+
+      <LayoutContent container={false} className="pb-0!">
```

Comment:
Skip to main content is missing from the very top (it used to appear visibly and be the first tabbable link on the homepage, program search, etc.)